docs/assets/app_component_dataset.js

```diff
@@ -213,8 +211,11 @@ const datasetView = () =>
                 c.dirs_from_path[c.dirs_from_path.length - 1]
                   .toLowerCase()
                   .indexOf(this.search_text.toLowerCase()) >= 0 ||
-                c.authors.some((f) =>f.givenName.toLowerCase().indexOf(this.search_text.toLowerCase()) >= 0 ) ||
-                c.authors.some((f) =>f.familyName.toLowerCase().indexOf(this.search_text.toLowerCase()) >= 0 )
+                // || (c.authors.some(e => e.givenName.toLowerCase().indexOf(this.search_text.toLowerCase()) >= 0))
+                c.authors.some(
+                  (f) =>
+                    f.name.toLowerCase().indexOf(this.search_text.toLowerCase()) >= 0
+                )
```

Comment:
Also not sure how this got reintroduced. Needs to be removed.

```suggestion
                c.authors.some((f) =>f.givenName.toLowerCase().indexOf(this.search_text.toLowerCase()) >= 0 ) ||
                c.authors.some((f) =>f.familyName.toLowerCase().indexOf(this.search_text.toLowerCase()) >= 0 )
```